frontend/src/features/mapSideBar/SidebarContents/subforms/IdentificationForm.tsx

```diff
@@ -0,0 +1,132 @@
+import './IdentificationForm.scss';
+
+import { FastSelect, FastInput, SelectOptions, Check } from 'components/common/form';
+import { Label } from 'components/common/Label';
+import AddressForm from 'features/properties/components/forms/subforms/AddressForm';
+import InformationForm from 'features/properties/components/forms/subforms/InformationForm';
+import LatLongForm from 'features/properties/components/forms/subforms/LatLongForm';
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+import { senstiveTooltip as sensitiveTooltip } from '../../../../../src/features/properties/components/forms/strings';
+import { HARMFUL_DISCLOSURE_URL } from 'constants/strings';
+
+interface IIdentificationProps {
+  agencies: SelectOptions;
+  classifications: SelectOptions;
+  predominateUses: SelectOptions;
+  constructionType: SelectOptions;
+  formikProps: any;
+  nameSpace?: any;
+  setMovingPinNameSpace: (nameSpace: string) => void;
+}
+
+export const IdentificationForm: React.FC<IIdentificationProps> = ({
```

Comment:
I know that a lot of this is due to my initial versions of these files not having comments, but let's try to have at least top level comments on new components going forward.